product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/index.mdx

```diff
@@ -16,11 +16,12 @@ You can modify your cluster by modifying:
 
 You can also modify your cluster by installing Postgres extensions. See [Postgres extensions](/biganimal/release/using_cluster/03_modifying_your_cluster/extensions) for more information.
 
-!!! Note
-    You can't switch from a single node cluster or a high availability cluster to an extreme high availability cluster or vice versa.
 
 ## Modify your cluster
 
+!!! Note
+    Any changes made to the cluster's instance type, volume type, or volume properties are not automatically applied to its replicas's settings. To avoid lag during replication, ensure that your replica is at least as capable as its parent cluster. See, 
```

Comment:
@moiznalwalla Is "replicas's" supposed to be singular or plural? if the former remove the s before the apostrophe, if the latter remove the ending s.

Seems that you were intending to add a cross reference. Either remove the "See," or add a cross reference if you think it would be useful.

Reply:
Also instead of saying "at least as capable" could you be more specific and mention that the instance/storage types should be at least as large as the source cluster?

Reply:
Fixed.